src/ufs.rs

```diff
@@ -62,16 +62,40 @@ impl Ufs {
 		if blkno < nd {
 			Ok(NonZeroU64::new(direct[blkno as usize] as u64))
 		} else if blkno < (nd + pbp) {
+			let x = blkno - nd;
+			let low = x % pbp;
```

Comment:
Isn't `low` guaranteed to be equal to `pbp` in this case?

Reply:
No, but it's guaranteed to be `< pbp`, so the `% pbp` is actually redundant.

Reply:
Err, that's right.  I meant < .

Reply:
Fixed